src/JTracker/Application.php

```diff
@@ -260,11 +260,10 @@ protected function doExecute()
 			// Log the error
 			$this->getLogger()->critical(
 				sprintf(
-					'Exception of type %1$s thrown with message %2$s',
-					get_class($exception),
-					$exception->getMessage()
+					'Exception of type %1$s thrown',
+					get_class($exception)
 				),
-				['trace' => $exception->getTraceAsString()]
+				['exception' => $exception]
```

Comment:
We don't use it explicitly, it was a way to show the trace for these uncaught Exceptions but that's rendered by Monolog as part of the exception context.